### PR TITLE
Validator for event start time 

### PR DESCRIPTION
**Pull request by reckocp**
Events can only start in the future

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -62,4 +71,9 @@ def get_remote_participation
   def get_total_participation
     invites.where(status: "Accepted").count + invites.where(status:"Accepted[remote]").count
   end
+
+  def future_date?
+    errors.add(:start_time, "can't be in the past") unless
+      start_time.future?
+  end
```
Comment (jah2488) [inline app/models/event.rb]: Ah, we're not using the `future_date?` method anymore now that we have the `FutureTime` validator, can you remove this method and repush?

Comment (reckocp) [inline app/models/event.rb]: Fixed.

**Comment by jah2488** [comment]
@reckocp You got one failing cucumber test, make that past and you've got a 👍  from me.

**Comment by Naghmehd** [comment]
👍

**Comment by reicht** [comment]
👍

**Pull request merged**